examples/exporting_for_inference.ipynb

```diff
@@ -62,102 +71,134 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
     "from torchvision.models import resnet\n",
     "from composer.models import ComposerClassifier\n",
-    "import composer.functional as cf\n",
+    "\n",
     "\n",
     "model = ComposerClassifier(module=resnet.resnet50())\n",
-    "model = cf.apply_squeeze_excite(model)\n",
     "\n",
-    "# switch to eval mode\n",
-    "model.eval()"
+    "\n"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "8f4bd162",
+   "id": "c2b15fb6",
    "metadata": {},
    "source": [
-    "Printing the model shows the new `Bottleneck` layers:"
+    "Now we create a callback that is used by the trainer to export model for inference. For this tutorial we are using `onnx` as export format but you can just change `save_format` from `save_format = 'onnx'` to `save_format = 'torchscript'` to export in the torchscript format. [ONNX](https://onnx.ai/) is a popular model format that can then be consumed by many third-party tools (e.g. TensorRT, OpenVINO) to optimize the model for specific hardware devices. \n",
+    "\n",
+    "Note: ONNX does not have a prebuild wheel for Mac M1/M2 chips yet, so is not pip installable. Skip this section if you are running on a Mac laptop."
```

Comment:
Would it be better to showcase TorchScript rather than ONNX for this tutorial, given ONNX's lack of support? It will simplify the tutorial a bit and reduce cognitive load on people going through it.

Reply:
The reason I went for onnx is because it is more trickier. We can go with torchscript or both.

Reply:
I see. I think it's better to start with the simpler way (TorchScript), you can then showcase ONNX

Reply:
We show export with torchscript at the top now.

Reply:
Great - thank you!